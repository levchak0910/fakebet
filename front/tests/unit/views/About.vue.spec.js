import {createWrapper, disableFile} from "../__utils__";
import Component from "@/views/About.vue";

disableFile();

describe("views/About.vue", () => {

    let wrapper;
    let mountWrapper = (options = {}) => {
        wrapper = createWrapper(Component, options);
    };

    it("Testing snapshot", () => {
        mountWrapper({
            computed: {
                leagueList: ["League 1", "League 2", "League 3"],
            },
        });

        expect(wrapper.element).toMatchSnapshot();
    });

});
