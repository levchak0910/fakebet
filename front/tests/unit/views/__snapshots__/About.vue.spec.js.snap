// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/About.vue Testing snapshot 1`] = `
<div
  id="about"
>
  <div>
    <b>
      Fakebet
    </b>
    <span>
       â€” site.description
    </span>
  </div>
  <div>
    <p>
      <b>
        site.forWho.question
      </b>
    </p>
    <p>
      site.forWho.answer
    </p>
  </div>
  <div>
    <p>
      <b>
        Data
      </b>
    </p>
    <p>
      site.dataFrom.coefficients:
      <ul>
        <li>
          <a
            href="https://www.parimatch.com"
          >
            Parimatch
          </a>
        </li>
        <li>
          <a
            href="http://www.williamhill.com"
          >
            Williamhill
          </a>
        </li>
        <li>
          <a
            href="https://www.leonbets.net"
          >
            Leonbets
          </a>
        </li>
      </ul>
    </p>
    <p>
      site.dataFrom.leagues:
      <ul>
        <li>
          League 1
        </li>
        <li>
          League 2
        </li>
        <li>
          League 3
        </li>
      </ul>
    </p>
    <p />
  </div>
  <div>
    <p>
      <b>
        site.notDo.
      </b>
    </p>
  </div>
  <div>
    <p>
      <b>
        Functional
      </b>
    </p>
    <p>
      site.functional
    </p>
  </div>
  <div>
    <p>
      <b>
        site.howMakeBet.question
      </b>
    </p>
    <ol>
      <li>
        site.howMakeBet.answer[0]
      </li>
      <li>
        site.howMakeBet.answer[1]
      </li>
      <li>
        site.howMakeBet.answer[2]
      </li>
      <li>
        site.howMakeBet.answer[3]
      </li>
      <li>
        site.howMakeBet.answer[4]
      </li>
      <li>
        site.howMakeBet.answer[5]
      </li>
      <li>
        site.howMakeBet.answer[6]
      </li>
    </ol>
  </div>
  <div>
    <span>
      site.openSource
    </span>
    <a
      href="https://github.com/levchak0910/fakebet"
    >
      github
    </a>
  </div>
</div>
`;
